### ISSUE TITLE
Horizon jobs won't run

### ISSUE THREAD

**Issue by ryancwalsh** [spatie/mailcoach-support#46]
/horizon/dashboard shows:
![image](https://user-images.githubusercontent.com/2086493/73968814-4e361080-48e8-11ea-94bd-764bfaf5921f.png)

/horizon/recent-jobs shows tons of jobs (mostly CalculateStatisticsJob but also SendCampaignJob), but they all say "-" for Runtime and have a Pause button for Status.
![image](https://user-images.githubusercontent.com/2086493/73973063-df5cb580-48ef-11ea-805d-65a26dabe478.png)

I then saw https://github.com/laravel/horizon/issues/279#issuecomment-408392856 and thought "Oh, I have more environments than just local and production, so this is probably the problem!"

But even after cloning the 'production' block from https://mailcoach.app/docs/package/general/installation-and-setup#install-horizon and calling it 'staging' and running `php artisan config:clear && php artisan queue:restart`, the Horizon dash looks the same.

Any ideas? Thanks.

**Comment by ryancwalsh** [comment]
I think I solved this problem.

I ran `sudo vim /etc/supervisor/conf.d/laravel-worker.conf` to go comment out the lines related to running `artisan queue:work --sleep=3 --tries=3 --daemon`, which I guess maybe were only relevant before Horizon?

Maybe now Horizon does _all_ of the queue working?

So then in MySQL I ran 
```
UPDATE `mailcoach_campaigns` SET `status`='draft', `scheduled_at`='2020-02-04 22:53:40' WHERE `id`='4';
```
And then my cron job sent the campaign.

and my Horizon dash looks like:

![image](https://user-images.githubusercontent.com/2086493/73974349-31063f80-48f2-11ea-9ab1-5dd46d65b6da.png)

Feel free to close this issue if this all sounds correct. Thanks!
